### PR TITLE
Expand info on quadplane tricopter yaw servo and lean angle param

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: plane/source/docs/guide-tilt-rotor.rst [hunk 1]
```diff
@@ -226,15 +226,14 @@ Non-Vectored Yaw
 
 Non-Vectored yaw aircraft (:ref:`Q_TILT_TYPE<Q_TILT_TYPE>` = 0 or 1) needs a tilt servo for yaw control.
 
-You need to setup your front tilt servos with ``SERVOn_FUNCTION=41``
-and also your servo for yaw control with ``SERVOn_FUNCTION=39``. You have to set up yaw servo’s maximum lean angle in degrees with :ref:`Q_M_YAW_SV_ANGLE<Q_M_YAW_SV_ANGLE>`.
+You need to setup your front tilt servos with ``SERVOn_FUNCTION=41`` and also your servo for yaw control with ``SERVOn_FUNCTION=39``, if the frame is a Tricopter. You should set up the yaw servo’s maximum lean angle in degrees with :ref:`Q_M_YAW_SV_ANGLE<Q_M_YAW_SV_ANGLE>`. This lean angle assumes that ``SERVOn_MIN`` and ``SERVOn_MAX``, represent =/- 90 degrees, with ``SERVOn_TRIM`` representing 0 degrees lean.
```
Comment (tridge) [inline plane/source/docs/guide-tilt-rotor.rst]: type, should be +/- 90 degrees

File: plane/source/docs/quadplane-frame-setup.rst [hunk 2]
```diff
@@ -71,6 +71,11 @@ layout
 -  3 for H frame
 -  11 for FireFly6Y6 (for Y6 only)
 
+Tricopter
+---------
+
+Frame Type 7 is Tricopter and can be either non-Tiltrotor configuration, or :ref:`Tiltrotor<guide-tilt-rotor>` configured using either Vectored or Non-Vectored yaw control. If using non-Tiltrotor or Non-Vectored Yaw Tilt-rotor, the yaw control output is setup as Motor 7 (``SERVOn_FUNCTION`` = 39) using a tilt mechanism for the yaw motor, Motor 4. You should set up the yaw servo’s maximum lean angle in degrees with :ref:`Q_M_YAW_SV_ANGLE<Q_M_YAW_SV_ANGLE>`. This lean angle assumes that ``SERVOn_MIN`` and ``SERVOn_MAX``, represent =/- 90 degrees, with ``SERVOn_TRIM`` representing 0 degrees lean.
```
Comment (tridge) [inline plane/source/docs/quadplane-frame-setup.rst]: +/- again

File: plane/source/docs/guide-tilt-rotor.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/guide-tilt-rotor.rst]: fixed

File: plane/source/docs/quadplane-frame-setup.rst (hunk 2 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-frame-setup.rst]: fixed

**Comment by Hwurzburg** [comment]
fixed ..merging

**Pull request merged**